var/spack/repos/builtin/packages/py-pycrypto/package.py

```diff
@@ -0,0 +1,40 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+#
+from spack import *
+
+
+class PyPycrypto(PythonPackage):
+    """The Python Cryptography Toolkit"""
+
+    homepage = "https://www.dlitz.net/software/pycrypto/"
+    url      = "https://pypi.python.org/packages/source/p/pycrypto/pycrypto-2.6.1.tar.gz"
+
+    version('2.6.1', '55a61a054aa66812daf5161a0d5d7eda')
+
+    # depends_on('py-setuptools', type=('build', 'link'))
+    # depends_on('py-setuptools', type='include')
+    # depends_on('gmp')
```

Comment:
Yes, it looks like this package requires either gmp or mpir for `Crypto.PublicKey._fastmath`. You don't have to add the latter, gmp is fine.